### ISSUE TITLE
SFTP: cannot download any files in OpenSSH log directory

### ISSUE THREAD

**Issue by jonradio** [PowerShell/Win32-OpenSSH#531]
None of the three .log files stored in the OpenSSH log directory can be downloaded using SFTP.  Using the Administrator username for all testing, these are the only files that I have found that I am not able to download.

My guess is that OpenSSH leaves the three log files Open in a state that effectively locks them against read or write operations.

Here is the result from FileZilla:
```
Command:	get "sftp-server.log" "C:\Users\jon\Desktop\sftp-server.log"
Error:	/C:/Program Files/OpenSSH-Win64/logs/sftp-server.log: open for read: received failure with description 'Failure'
Error:	File transfer failed
```
Environment:
Windows Server 2016
OpenSSH-Win64.zip
v0.0.8.0

Client Software: FileZilla on Windows 10

**Comment by joeyaiello** [comment]
I can confirm that repros to/from 0.0.9.0. Interestingly, even a remote `ls` doesn't show the files in the `logs` directory:

```
sftp> pwd
Remote working directory: /C:/OpenSSH-Win64
sftp> ls
install-sshd.ps1            install-sshlsa.ps1          logs                        ntrights.exe
null                        scp.exe                     sftp-server.exe             sftp.exe
ssh-add.exe                 ssh-agent.exe               ssh-agent.log               ssh-keygen.exe
ssh-lsa.dll                 ssh-shellhost.exe           ssh.exe                     ssh_host_dsa_key
ssh_host_dsa_key.pub        ssh_host_ecdsa_key          ssh_host_ecdsa_key.pub      ssh_host_ed25519_key
ssh_host_ed25519_key.pub    ssh_host_rsa_key            ssh_host_rsa_key.pub        sshd.exe
sshd.log                    sshd_config                 uninstall-sshd.ps1          uninstall-sshlsa.ps1
sftp> cd logs
sftp> ls
sftp>
```

In actuality that `logs` folder has three log files in it.

**Comment by bagajjal** [comment]
Fixed in the june mid release (0.0.16.0, yet to be released).
